### PR TITLE
[55] Add Page Navigation to your pages 

### PR DESCRIPTION
**Pull request by queensaldam**
## Task: `Added navigation to the pages`
- `Navigation to product & contact page was added

### CODE REVIEW

File: submissions/damilolaSalami/html/productpage.html [hunk 1]
```diff
@@ -11,10 +11,10 @@
         <h1><a href="#">BuyMe</a></h1>
         <nav>
             <ul>
-                <li><a href="#">LogIn</a></li>
+                <li><a href="#">Log In</a></li>
                 <li><a href="#">Create a profile</a></li>
-                <li><a href="#">Cart</a></li>
-                <li><a href="#">Contact us</a></li>
+                <li><a href="../html/productpage.html"></a>Product page</li>
```
Comment (adaugochi) [inline submissions/damilolaSalami/html/productpage.html:16]: simply do `href="productpage.html"`. Also, what about your contact form page? Are you not linking pages on that too?

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/damilolaSalami/html/contact.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/damilolaSalami/html/contact.html:1]: This should be inside `html-forms` and should have been done prior to now

File: submissions/damilolaSalami/html/contact.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+</head>
+
+<body>
+    <header>
+        <h1><a href="#">BuyMe</a></h1>
+        <nav>
+            <ul>
+                <li><a href="#">LogIn</a></li>
+                <li><a href="#">Create a profile</a></li>
+                <li><a href="productpage.html">Product Page</a></li>
```
Comment (adaugochi) [inline submissions/damilolaSalami/html/contact.html:16]: You are only expected to add navigation links to tat already created page

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/damilolaSalami/html/productpage.html [hunk 4]
```diff
@@ -11,10 +11,10 @@
         <h1><a href="#">BuyMe</a></h1>
         <nav>
             <ul>
-                <li><a href="#">LogIn</a></li>
+                <li><a href="#">Log In</a></li>
                 <li><a href="#">Create a profile</a></li>
-                <li><a href="#">Cart</a></li>
-                <li><a href="#">Contact us</a></li>
+                <li><a href="productpage.html">Product page</a></li>
+                <li><a href="contact.html">Contact us</a></li>
```
Comment (adaugochi) [inline submissions/damilolaSalami/html/productpage.html:17]: Does this link work?

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by adaugochi** [comment]
I checked your work and I noticed you haven't created contact form page before

You ought to do that before doing this task.

**Comment by queensaldam** [comment]
> I checked your work and I noticed you haven't created contact form page before
> 
> You ought to do that before doing this task.

Thank you. I'm sorry for all the troubles.
Please am I to create a PR branch on html form containing contact page first? Then update html branch with the navigation page?

**Comment by ToluAkin** [comment]
@queensaldam In this case, you can have **2 PRs that will not conflict.** You can create a new pull request with the task_id for the contact form only. If you have any issues, you can reach out. Remember, you create a new branch from the master branch.

**Comment by queensaldam** [comment]
> > @queensaldam In this case, you can have **2 PRs that will not conflict.** You can create a new pull request with the task_id for the contact form only. If you have any issues, you can reach out. Remember, you create a new branch from the master branch.
> 
> Thank you

I've done it

**Comment by adaugochi** [comment]
We have said it over time. "Don't have 2 PRs open at the same time". Some of these changes you have on this PR are still in the previous PR. Kindly ensure that one PR is merged first before creating another one to avoid confusion from your end. 

Going forward, you have to close this PR for now. Fix the comments on your previous PR, get it merged then proceed to create another one. Thanks. If you don't understand what I have said, kindly reach out

**Pull request closed without merging**